kasa/iot/modules/motion.py

```diff
@@ -30,6 +93,11 @@ def _initialize_features(self) -> None:
         if "get_config" not in self.data:
             return
 
+        # Require that ADC value is also present.
+        if "get_adc_value" not in self.data:
```

Comment:
Is the expectation that all PIR devices will have `get_adc_value` or could some only have `get_config` causing this line to impact those devices?

Reply:
I expect that all PIR devices will have the `get_adc_value`, but of course I have no way of actually proving that for future devices. It's possible an updated model could come out with a digital/event driven PIR sensor instead, but then I would expect the `get_config` to take on a different form than what is currently supported as well.

I can remove the check if that's better for device support.